linters/tslint.json

```diff
@@ -0,0 +1,154 @@
+{
+  "extends": ["tslint-eslint-rules"],
+  "rules": {
+
+    // RECOMMENDATIONS
+
+    "interface-name": [true, "never-prefix"],     // Require interface names to not have an “I” prefix
```

Comment:
I think we should have the `I` prefix.  We tend to name things a certain way. For example we append `Controller`  `Service`, `Model` , etc to our files. 

```
ApplicationController
ProductService
ProductModel
```
Its a nice way to self document or at a quick glance see what type of file it is. e.g. `IProduct` & `ProductModel`

Reply:
@ekidd

Reply:
I agree, but most voted against the "I".

Reply:
@codeBelt I don't disagree with the idiom - I also understand that this is a convention in C#. I also believe that an implementer should be able to choose idioms that make the most sense for their situation (the examples you provided of names like `ApplicationController` are conventions, and not standardized).

I'm leaning towards flexibility - perhaps there shouldn't be a standard here, let the implementer choose.